docs/binary.md

```diff
@@ -58,7 +67,7 @@ It is just like counting in decimal except we reach 10 much sooner.
 | Decimal       | Explanation            |
 |:-------------:|:----------------------:|
 |	99	 |	When we run out of digits, we ...|
-|	100	 |	... start back at 0 again, but add 1 on the left|
+|	100	 |	... start back at 0 again, but carry 1 on the left|
```

Comment:
Make this `but carry 1 on the left` bold as done in other tables

Reply:
made 0 and 1 bold as a where required